### PR TITLE
feat(snippets): add Oneko snippet

### PR DESCRIPTION
**Pull request by somerandomcloud**
Adds a sleeping cat on the music progress bar. Adapted from a Vencord snippet created by @MeguminSama and modified to work on Spicetify by me!

[Screencast from 28-05-23 02:51:17.webm](https://github.com/spicetify/spicetify-marketplace/assets/39058370/8c757069-a90f-48ae-81a2-e5024b662078)

### CODE REVIEW

File: src/resources/snippets.ts [hunk 1]
```diff
@@ -197,4 +197,10 @@ export default [
...
     "preview": "resources/assets/snippets/pretty-lyrics.png",
   },
+  {
+    "title": "Oneko",
+    "description": "Adds Oneko onto your playback bar!",
+    "code": "[class^='playback-']:not(.volume-bar__slider-container > .playback-progressbar):before { content: ''; width: 32px; height: 32px; bottom: calc(100% - 7px); right: 10px; position: absolute; image-rendering: pixelated; background-image: url('https://raw.githubusercontent.com/adryd325/oneko.js/14bab15a755d0e35cd4ae19c931d96d306f99f42/oneko.gif'); animation: oneko 1s infinite; } @keyframes oneko { 0%, 50% { background-position: -64px 0; } 50.0001%, 100% { background-position: -64px -32px; } }",
+    "preview": "Preview image of the snippet"
```
Comment (kyrie25) [inline src/resources/snippets.ts:204]: Provide a proper URL for the image preview

**Review by kyrie25 — CHANGES_REQUESTED** [review]

File: src/resources/snippets.ts (hunk 1 above)
Comment (somerandomcloud) [reply src/resources/snippets.ts]: Whoops, forgot to edit that. Should be fixed

File: src/resources/snippets.ts [hunk 2]
```diff
@@ -204,6 +204,10 @@ export default [
     "preview": "resources/assets/snippets/pretty-lyrics.png",
   },
   {
+    "title": "Oneko",
+    "description": "Adds Oneko onto your playback bar!",
+    "code": "[class^='playback-']:not(.volume-bar__slider-container > .playback-progressbar):before { content: ''; width: 32px; height: 32px; bottom: calc(100% - 7px); right: 10px; position: absolute; image-rendering: pixelated; background-image: url('https://raw.githubusercontent.com/adryd325/oneko.js/14bab15a755d0e35cd4ae19c931d96d306f99f42/oneko.gif'); animation: oneko 1s infinite; } @keyframes oneko { 0%, 50% { background-position: -64px 0; } 50.0001%, 100% { background-position: -64px -32px; } }",
+    "preview": "resources/assets/snippets/oneko.png"
     "title": "Remove Popular sections from homepage",
```
Comment (theRealPadster) [inline src/resources/snippets.ts:211]: It looks like I messed it up a bit in the merge. I think this should fix it?

```suggestion
    "preview": "resources/assets/snippets/oneko.png",
  },
  {
    "title": "Remove Popular sections from homepage",
```

**Review by kyrie25 — CHANGES_REQUESTED** [review]
Resolve build error

**Pull request merged**